tests/integration/conftest.py

```diff
@@ -86,7 +89,48 @@ async def ingress_requirer_application_fixture(
     application = await model.deploy(
         ingress_requirer_application_name, channel="latest/edge", constraints="mem=1024M"
     )
-    await model.wait_for_idle(apps=[ingress_requirer_application_name], status="active")
+    return application
+
+
+@pytest_asyncio.fixture(scope="module", name="any_charm_ingress_requirer")
+async def any_charm_ingress_requirer_fixture(model: Model):
+    """Deploy any-charm and patch it with ingress lib."""
+    any_app_name = "any-ingress"
+    ingress_lib_url = (
+        "https://github.com/canonical/nginx-ingress-integrator-operator"
+        "/raw/main/lib/charms/nginx_ingress_integrator/v0/ingress.py"
+    )
```

Comment:
This is not the default location of the ingress library which is there:
`https://raw.githubusercontent.com/canonical/charm-relation-interfaces/main/lib/charms/interfaces/v2/ingress.py`

But isn't there a way to call fetch-lib instead ? I don't know if juju or pytest_operator provide this feature

Reply:
I haven't seen `fetch-libs` calls in pytest_operator, I fixed the location of the library for now